README.md

```diff
@@ -75,32 +75,55 @@ See <a href="https://semgrep.dev/docs/supported-languages/">supported languages<
 
 ### Getting started
 
-To install Semgrep use Homebrew or pip, or run without installation via Docker:
+1.  Register to <a href="http://bit.ly/424Jbs0" target="_blank">semgrep.dev</a>
```

Comment:
The use of a bit.ly link here feels out of place. As a security professional I am hesitant to click on such links, and would prefer to either see a semgrep.dev domain or all of the details of the utm/campaign.